closure/goog/i18n/numberformat.js

```diff
@@ -462,9 +462,9 @@ goog.i18n.NumberFormat.prototype.parseNumber_ = function(text, pos) {
       normalizedText += '.';
       sawDecimal = true;
     } else if (ch == grouping.charAt(0) &&
-               ('\u00a0' != grouping.charAt(0) ||
-                pos[0] + 1 < text.length &&
-                this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
+        ('\u00a0' != grouping.charAt(0) ||
+        pos[0] + 1 < text.length &&
+        this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
```

Comment:
The indentation seems weird. I think the original was correct. Can you double check?